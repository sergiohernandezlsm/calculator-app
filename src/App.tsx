import 'bootstrap/dist/css/bootstrap.min.css';
import CalculatorPage from './pages/calculator-page';

function App() {
  return (
    <CalculatorPage />
  );
}

export default App;
