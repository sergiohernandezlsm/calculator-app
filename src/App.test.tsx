import { shallow } from 'enzyme';
import App from './App';

test('renders react App component', () => {
  const wrapper = shallow(<App />);
  expect(wrapper).toMatchSnapshot();
});
