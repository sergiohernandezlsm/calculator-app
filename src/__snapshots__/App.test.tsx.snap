// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders react App component 1`] = `ShallowWrapper {}`;
